antiplag/views.py

```diff
@@ -22,7 +22,8 @@ def post(self, request):
 
         # create new submission
         submission = Submission.objects.create(
-            status=Submission.SubmissionStatus.PENDING
+            status=Submission.SubmissionStatus.PENDING,
+            email = request.POST.get("email", "")
```

Comment:
So in case the email field is empty, the default value is empty string? Why are we having null=True in the database? I think having null in that case would be better than empty string.